tests/scripts/run-xunit-perf.cmd

```diff
@@ -73,27 +71,37 @@ setlocal
   set BENCHDIR=%~p1
 
   rem copy benchmark and any input files
-  call :run_cmd xcopy /s %~1 . >> %RUNLOG%  || exit /b 1
+  call :run_cmd xcopy /sy %~1 . >> %RUNLOG%  || exit /b 1
   if exist "%BENCHDIR%*.txt" (
-    call :run_cmd xcopy /s %BENCHDIR%*.txt . >> %RUNLOG%  || exit /b 1
+    call :run_cmd xcopy /sy %BENCHDIR%*.txt . >> %RUNLOG%  || exit /b 1
   )
 
-  set CORE_ROOT=%CORECLR_REPO%\sandbox
-
   rem setup additional environment variables
   if DEFINED TEST_ENV (
     if EXIST "%TEST_ENV%" (
         call "%TEST_ENV%"
     )
   )
 
-  set BENCHNAME_LOG_FILE_NAME=%BENCHNAME%.log
+  echo/
+  echo/  ----------
+  echo/  Running %BENCHNAME%
+  echo/  ----------
+
+  set LV_RUNID=Perf-%ETW_COLLECTION%
+  set BENCHNAME_LOG_FILE_NAME=%LV_RUNID%-%BENCHNAME%.log
+  set LV_CMD=
   if defined IS_SCENARIO_TEST (
-    call :run_cmd corerun.exe "%CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT%" --perf:runid Perf 1>"%BENCHNAME_LOG_FILE_NAME%" 2>&1
+    set "LV_CMD=corerun.exe "%CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT%" --perf:runid "%LV_RUNID%""
   ) else (
-    call :run_cmd %STABILITY_PREFIX% corerun.exe PerfHarness.dll "%CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT%" --perf:runid Perf --perf:collect %COLLECTION_FLAGS% 1>"%BENCHNAME_LOG_FILE_NAME%" 2>&1
+    set "LV_CMD=%STABILITY_PREFIX% corerun.exe PerfHarness.dll "%CORECLR_REPO%\sandbox\%BENCHNAME%.%TEST_FILE_EXT%" --perf:runid "%LV_RUNID%" --perf:collect %COLLECTION_FLAGS%"
   )
 
+  echo/
+  echo/%USERNAME%@%COMPUTERNAME% "%CD%"
+  echo/[%DATE%][%TIME:~0,-3%] $ !LV_CMD!
```

Comment:
It looks like `:run_cmd` already prints this out. Doesn't this cause us to just print everything twice?